acos_client/v30/slb/virtual_port.py

```diff
@@ -109,8 +109,10 @@ def _set(
             params['port']['template-virtual-port'] = virtual_port_templates.get('template-virtual-port', None)
             if protocol in ['http', 'https']:
                 params['port']['template-http'] = virtual_port_templates.get('template-http', None)
-            else:
+            elif protocol in ['tcp', 'TCP']:
                 params['port']['template-tcp'] = virtual_port_templates.get('template-tcp', None)
+            else
```

Comment:
Invalid syntax for `else`

Reply:
Added ":" after else statement in newest commit.